modules/ROOT/pages/access-logging.adoc

```diff
@@ -142,3 +142,5 @@ Based on this setting, the NCSA access log will include the following informatio
 * The first line of the request
 * The status code of the response
 * The response size in bytes excluding headers
+* The elapsed time of the request in milliseconds (end-to-end including client and network time)
```

Comment:
revise to remove parentheses per ID guidelines:

The elapsed time of the request in milliseconds, end-to-end, including client and network time

Reply:
I also just realized that the value is actually in microseconds (although millisecond accuracy), so I'll change milliseconds to microseconds.